theme.js

```diff
@@ -2944,4 +2944,15 @@ export default (variables = defaultThemeVariables) => ({
       borderWidth: 1,
     },
   },
+
+  'shoutem.ui.AnimatedLoading': {
+    container: {
+      width: getSizeRelativeToReference(44, 375, window.width),
```

Comment:
since this is a generic component, we don't want to have fixed container dimensions. The idea is to wrap anything under this, and it will work. What would happen here if you had a child that is larger than these dimensions ? use onLayout or it's equivalent to get children dimensions. Apply the same to the animation container, and then position and scale the animation in the same container

Reply:
I've applied these changes and I'm strongly against it, we can discuss when you'll be able to.
This component will sometimes be used without children, like we do in social now.
When it is used without children, LoadingContainer flexes to parent, that can be full screen e.g., while children container was only 100 px height... (hard to explain like this :D)
The fact is, it can break layout and I'm strongly against defining View inside LoadingContainer only to adjust its dimensions properly.

On the other hand, code that I've had and you commented here is much more what we already do everywhere in Shoutem.
Those "unused" styles you mentioned - those are for users to override and set their desired dimensions for this container.
These default dimensions (minWidth and minHeight now) only assure  loading container to be enough width and height to show loading.
If user wants container to be specific dimensions - user will pass `style={{ container: {width: 300, height: 300} }}` to this component. Animation container will have loading animation centered and flexed.
If loading animation container is too big because it flexed, user will then pass style.loading - again, it'll be centered.

example:
```
style={{
              container: {
                width: 300,
                height: 300,
              },
              loading: {
                width: 100,
                height: 100,
              },
```

<br><br>
**Video:**

https://user-images.githubusercontent.com/57353746/156996062-3a46f8a8-a457-4033-90dd-cac80ddcc81b.mov

Reply:
Ignore my previous comment.
With Slavko's help, I've applied suggested changes, component will now have dimensions per children layout dimensions.